src/modules/clearItems.js

```diff
@@ -1,11 +1,11 @@
-/* eslint-disable linebreak-style */
-import Todolist, {
-  list,
-} from './listClass.js';
-
-export default function checkIfDone() {
-  const listFiltered = list.filter((obj) => obj.complete !== true);
-  localStorage.setItem('listStorage', JSON.stringify(listFiltered));
-  Todolist.createList();
-  window.location.reload();
-}
+/* eslint-disable linebreak-style */

```

Comment:
Implementing linters is enforced to make our code clean and follow industry standards. I think instead of disabling linters, finding ways to solve linter errors would be a much better solution.